package.json

```diff
@@ -1588,15 +1573,12 @@
     "geckodriver": "^4.3.3",
     "gulp-brotli": "^3.0.0",
     "gulp-postcss": "^9.0.1",
-    "gulp-sourcemaps": "2.6.5",
     "gulp-terser": "^2.1.0",
     "has-ansi": "^3.0.0",
-    "hdr-histogram-js": "^1.2.0",
```

Comment:
This seems to be used at `packages/kbn-apm-synthtrace/src/lib/utils/create_lossless_histogram.ts`

Reply:
I thought this was using `native-hdr-histogram`, as opposed to `hdr-histogram-js` (via https://github.com/elastic/kibana/pull/147036). @dgieselaar can you help us understand if `hdr-histogram-js` is used somewhere?

Reply:
oh, that embarassing.  I didn't search correctly.  Thanks for clarifying!

Reply:
Thanks for digging into the PR, I appreciate the review!